### ISSUE TITLE
Bug: Can't publish apk

### ISSUE THREAD

**Issue by BugsBunnyBR** [cianru/huawei-publish-gradle-plugin#33]
I just tried the release 1.3.0 and the sample project here...
``` shell
 ./gradlew assembleDemoRelease
 ./gradlew publishHuaweiAppGalleryDemoRelease
```

 Build file /Users/julio.buenocotta/git/huawei-publish-gradle-plugin/sample-groovy/build/outputs/apk/demo/release has wrong file extension that doesn't match with announced buildFormat(APK) plugin extension param.

**Comment by BugsBunnyBR** [comment]
@cosic are you there ?

**Comment by cosic** [comment]
@BugsBunnyBR Hi! Yes, I'm here. I'm going to  figure out it next weekend.

**Comment by cosic** [comment]
I found a couple issues which blocked me https://github.com/gradle/gradle/issues/16777 and https://github.com/gradle/gradle/issues/16775. I'm trying to figure it out.

**Comment by BugsBunnyBR** [comment]
For the first issue...
have you tried this workaround https://github.com/gradle/gradle/issues/16777#issuecomment-817303964 ?
It is odd, someone commented that is was fixed in gradle 7.1.1... and ktlint was able to workaround it as well... maybe looking at the code or their changelog would help ?

**Comment by cosic** [comment]
@BugsBunnyBR Sorry for long reaction. But seems I fixed this issue. Could you check `1.3.1-SNAPSHOT` version or into `snapshot-1.3.1` branch?

**Comment by cosic** [comment]
@BugsBunnyBR Hi! Could you check again `1.3.1-SNAPSHOT` version please?

**Comment by cosic** [comment]
@BugsBunnyBR Is `/builds/veepee/offerdiscovery/products/front-mobile/android/android-vente-privee/app/build/outputs/apk/veepeeAll/release` correct path for build `apk` file folder? Does it contain any `apk` file?

**Comment by BugsBunnyBR** [comment]
It seems that compiling the app in a different command resulted in the upload task to work!
This works
```
./gradlew clean assembleRelease
./gradlew :app:publishHuaweiAppGalleryRelease
```

This doesn't work
```
./gradlew clean assembleRelease :app:publishHuaweiAppGalleryRelease
```

**Comment by cosic** [comment]
@BugsBunnyBR Yes i got the same feedback [here (issue#26)](https://github.com/cianru/huawei-publish-gradle-plugin/issues/26) but haven't tackled the issue yet. Do you have any idea why?

**Comment by cosic** [comment]
Fixed at [v1.3.1](https://github.com/cianru/huawei-publish-gradle-plugin/releases/tag/v1.3.1)
